templates/events-header.php

```diff
@@ -30,7 +30,7 @@
 			<?php endif; ?>
 		<?php endif; ?>
 	</ul>
-	<?php if ( isset( $event ) && ! isset( $event_form_name ) ) : ?>
+	<?php if ( isset( $event ) && ! isset( $event_form_name ) && ! isset( $hide_sub_head ) ) : ?>
```

Comment:
In a follow-up PR, I want to introduce the `$hide_sub_head` flag to the forms. That way we won't need to check for this line would become
```suggestion
	<?php if ( isset( $event ) && && ! isset( $hide_sub_head ) ) : ?>
```

Reply:
Makes sense. Maybe the flag could be called `$show_sub_head`, to prevent a double negative, which I find harder to reason about.